django/contrib/gis/db/backends/spatialite/operations.py

```diff
@@ -246,6 +259,16 @@ def spatial_aggregate_name(self, agg_name):
         agg_name = 'unionagg' if agg_name.lower() == 'union' else agg_name.lower()
         return getattr(self, agg_name)
 
+    def spatial_function_name(self, func_name):
+        geom_func_map = {
+            'Length': 'ST_Length',
+            'Reverse': 'ST_Reverse',
+            'Scale': 'ScaleCoords',
+            'Translate': 'ST_Translate',
+            'Union': 'ST_Union',
+        }
```

Comment:
Is there a reason this can't be a class attribute with a default implementation in the `BaseSpatialOperations` class?